deploy/environments/applications.yaml

```diff
@@ -164,4 +166,4 @@ spack:
     - synapsetool
     - touchdetector
     - ultraliser
-    - unit-test-translator
\ No newline at end of file
+    - unit-test-translator
```

Comment:
Not sure what this is, perhaps some temporary thing someone added back in the day? Not mine, tbh.

Reply:
Certain people don't have their editor configured right :angry: